src/test/java/io/pravega/connectors/flink/FlinkPravegaTableFactoryTest.java

```diff
@@ -0,0 +1,272 @@
+/**
+ * Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.connectors.flink;
+
+import io.pravega.connectors.flink.FlinkPravegaTableSourceTest.TestTableDescriptor;
+import io.pravega.client.stream.Stream;
+import org.apache.flink.table.api.NoMatchingTableFactoryException;
+import org.apache.flink.table.api.Types;
+import org.apache.flink.table.api.ValidationException;
+import org.apache.flink.table.descriptors.DescriptorProperties;
+import org.apache.flink.table.descriptors.Json;
+import org.apache.flink.table.descriptors.Schema;
+import org.apache.flink.table.factories.StreamTableSinkFactory;
+import org.apache.flink.table.factories.StreamTableSourceFactory;
+import org.apache.flink.table.factories.TableFactoryService;
+import org.apache.flink.table.sinks.TableSink;
+import org.apache.flink.table.sources.TableSource;
+import org.apache.flink.table.sources.TableSourceUtil;
+import org.junit.Test;
+
+import java.net.URI;
+import java.util.HashMap;
+import java.util.Map;
+
+import static io.pravega.connectors.flink.Pravega.CONNECTOR_WRITER_MODE;
+import static org.junit.Assert.assertNotNull;
+
+/**
+ * Unit test that validates configurations that can be passed to create source and sink factory.
+ */
+public class FlinkPravegaTableFactoryTest {
+
+    final static Schema SCHEMA = new Schema()
+                                            .field("name", Types.STRING() )
+                                            .field("age", Types.INT() );
+    final static Json JSON = new Json().failOnMissingField(false) .deriveSchema();
+    final static String SCOPE = "foo";
+    final static String STREAM = "bar";
+    final static String CONTROLLER_URI = "tcp://localhost:9090";
+
+    final static PravegaConfig PRAVEGA_CONFIG = PravegaConfig.fromDefaults()
+                                                            .withControllerURI(URI.create(CONTROLLER_URI))
```

Comment:
Continuous indention is not consistent. 
(Consistent code style check might be necessary on Jenkins level.)

Reply:
fixed